src/components/Swap/Summary/Details.tsx

```diff
@@ -69,14 +67,14 @@ export default function Details({ trade, slippage, gasUseEstimateUSD, impact }:
     }
 
     if (gasUseEstimateUSD) {
-      rows.push([t`Network fee`, `~$${gasUseEstimateUSD.toFixed(2)}`])
+      rows.push([t`Network fee`, `~${formatCurrencyAmount(gasUseEstimateUSD, true)}`])
```

Comment:
I think you no longer want the `~`.

Reply:
we need it... it's saying "approximate" network fee